src/Components/Admin/Sessions/Form/index.js

```diff
@@ -96,6 +96,22 @@ function sessionsForm() {
     return value ? undefined : 'Required';
   };
 
+  const validateDate = (value) => {
+    if (required(value)) {
+      return 'Required';
+    }
+    let sessionDate = value.split('T');
+    let dateValue = Math.round(new Date(sessionDate[0]).getTime() / 1000);
+
+    const now = new Date();
+    const year = now.getFullYear();
+    const month = now.getMonth() + 1 < 10 ? `0${now.getMonth() + 1}` : now.getMonth() + 1;
+    const date = now.getDate() < 10 ? `0${now.getDate()}` : '';
+
+    let nowValue = Math.round(new Date(`${year}-${month}-${date}`).getTime() / 1000);
+    return dateValue >= nowValue ? undefined : 'Invalid date';
+  };
+
```

Comment:
This validation should only work when you are creating a new one, not when editing.

Reply:
done